src/configuration/Configuration.cpp

```diff
@@ -156,34 +159,34 @@ namespace SlimeVR {
         void Configuration::loadCalibrations() {
 #ifdef ESP32
             {
-                File calibrations = LittleFS.open("/calibrations");
+                File calibrations = LittleFS.open(DIR_CALIBRATIONS);
                 if (!calibrations) {
                     m_Logger.warn("No calibration data found, creating new directory...");
 
-                    if (!LittleFS.mkdir("/calibrations")) {
-                        m_Logger.error("Failed to create directory: /calibrations");
+                    if (!LittleFS.mkdir(DIR_CALIBRATIONS)) {
+                        m_Logger.error("Failed to create directory: " DIR_CALIBRATIONS);
                         return;
                     }
 
-                    calibrations = LittleFS.open("/calibrations");
+                    calibrations = LittleFS.open(DIR_CALIBRATIONS);
                 }
 
                 if (!calibrations.isDirectory()) {
                     calibrations.close();
 
-                    m_Logger.warn("Found file instead of directory: /calibrations");
+                    m_Logger.warn("Found file instead of directory: " DIR_CALIBRATIONS);
```

Comment:
@unlogisch04 this is an implicit string literal concatenation - it should work fine, if you were concerned.
https://www.techiedelight.com/concatenate-string-literals-c-cpp/

Reply:
thx